Content.Server/_RMC14/Xenonids/Hive/XenoHiveSystem.cs

```diff
@@ -54,7 +66,19 @@ public override void Update(float frameTime)
                 continue;
 
             var popup = $"The Hive can now support: {string.Join(", ", _announce)}";
-            _xenoAnnounce.AnnounceSameHive(default, popup, hive.AnnounceSound, PopupType.Large);
+            // below is commented out because it used to not do anything, now it works but isnt localized so i think its best to have a dedicated pr look at it
+            //_xenoAnnounce.AnnounceTohive(hiveId, popup, hive.AnnounceSound, PopupType.Large);
```

Comment:
It's fine to have it unlocalized for now, I just forgot to fix it
```suggestion
            _xenoAnnounce.AnnounceTohive(hiveId, popup, hive.AnnounceSound, PopupType.Large);
```